keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/TokenHolder.kt

```diff
@@ -17,11 +18,17 @@ internal class TokenHolder(private val client: KeycloakLoginClient,
         val LOG = LoggerFactory.getLogger(TokenHolder::class.java)!!
     }
 
+    private val safetyDurationMs = 1000
```

Comment:
I would like to see this variable as a parameter with the default value of `1000`

Reply:
done